src/Umbraco.Cms.Integrations.Search.Algolia/Services/AlgoliaSearchPropertyIndexValueFactory.cs

```diff
@@ -29,11 +34,11 @@ public AlgoliaSearchPropertyIndexValueFactory(IDataTypeService dataTypeService,
         {
             var dataType = _dataTypeService.GetByEditorAlias(property.PropertyType.PropertyEditorAlias)
                 .FirstOrDefault(p => p.Id == property.PropertyType.DataTypeId);
-
+         
             if (dataType == null) return default;
 
-            var indexValues = dataType.Editor.PropertyIndexValueFactory.GetIndexValues(property, culture, string.Empty, true);
-
+            var indexValues = dataType.Editor.PropertyIndexValueFactory.GetIndexValues(property, culture, string.Empty, true, _availableLanguages);
```

Comment:
Makes sense to use the non-deprecated method, but I don't understand the fix I'm afraid.  Looking in the CMS code, at `RichTextPropertyIndexValueFactory`, which was mentioned in the issue as being one that causes the problem, I can't see where the `availableLanguages` parameter is used.  And as such why passing in `Enumerable.Empty<string>` isn't fine here.

Probably I'm missing something - but if we do need to move the call to retrieve to all languages here, it's potentially an expensive thing to do so wanted to make sure it is necessary.